.gitignore

```
@@ -1,4 +1,5 @@
 /mysql/data/
 /.idea/
 FinanSu.iml
-finansu.env
\ No newline at end of file
+finansu.env
+go_api/tmp
```

```suggestion
api/tmp
```
おそらくlotteryを見て`go_api/tmp`にしたと思うのですが、今回のFinanSuのディレクトリ構造で、tmpができるのは`api`下なので`api/tmp`となります。